operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -117,6 +117,13 @@ mimir:
       type: enterprise
     {{- end }}
 
+    frontend_worker:
```

Comment:
There is already a `frontend_worker` block further down. Can you move the changes here there?